vw_example_builder.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List, Optional
+from sentence_transformers import SentenceTransformer
+
+
+class Embedder(ABC):
+    @abstractmethod
+    def to_vw_format(self, **kwargs) -> str:
+        pass
+
+
+class ContextualBanditTextEmbedder(Embedder):
```

Comment:
probably for future PRs:
not sure about statefullness of this guy:
In order to use it properly we need to do sequence of 3 calls:
embed_actions(actions)
embed_context(context)
to_vw_format()

Although both embed_actions and embed_context are not doing much CB-specific stuff, but just iterating over feature containers. The only CB-specific piece of logic seems to be default namespacing which can be provided as input argument - something like having method outside of the class:
def embed(Union[str, List[string], List[dict]], default_namespace) -> Union[Dict, List]
   ...
can both reduce potential misusage of the class and reduce amount of code.